cocaine12/worker.go

```diff
@@ -260,7 +224,7 @@ func (w *Worker) isStopped() bool {
 	return false
 }
 
-func (w *Worker) loop() error {
+func (w *Worker) Loop() error {
```

Comment:
Is it needed to make it public? I found existance of Run() and Loop() inconvenient